server/app.py

```diff
@@ -15,7 +15,7 @@
 
 app = Flask(__name__, static_url_path='',
             static_folder=Constants.STATIC_FOLDER)
-api = Api(app)
+api = Api(app, no_doc=True)
```

Comment:
can you delete the `no_doc=True` here? I think that will get rid of the swagger.